xpk.py

```diff
@@ -1598,6 +2073,51 @@ def enable_kueue_crds(args, system) -> int:
   return 0
 
 
+def get_kueue_covered_resources_config(args, cluster_hardware_name, resource_type, total_chips) -> str:
```

Comment:
Just to note this might conflict with pathways changes (https://github.com/google/xpk/pull/74/files). Not sure which PR will go in first but I am happy to work through the merge conflict.

I like this function so hopefully we can use this in pathways PR. cc @RoshaniN

Reply:
Looks like Pathways PR is merged already. How do we proceed here for any potential conflict?

Reply:
I made a helper to add resources to kueue config for Pathways, we can work on adding the kueue config conditionally.

If --enable-pathways, {pathways resources are added}
if GPU , {GPU resources are added}